src/pull-request/pull-request-diff.html

```diff
@@ -1,9 +1,17 @@
 <link rel="import" href="../bower_components/iron-icons/iron-icons.html">
 <link rel="import" href="../bower_components/paper-icon-button/paper-icon-button.html">
 <link rel="import" href="../bower_components/paper-toolbar/paper-toolbar.html">
-<link rel="import" href="pull-request-chunk.html">
+<link rel="import" href="../bower_components/prism-element/prism-highlighter.html">
+<link rel="import" href="../bower_components/prism-element/prism-style-import.html">
 <link rel="import" href="../ajax/authenticated-ajax.html">
+<link rel="import" href="pull-request-chunk.html">
 
+<script src="../bower_components/prism/components/prism-diff.min.js"></script>
+// Match all lines that are not insertions or deletions
+Prism.languages.diff.other = /.+/m;
+// Additions and deletions can also be an empty line
+Prism.languages.diff.inserted = /^[+>].*$/m
+Prism.languages.diff.deleted = /^[-<].*$/m
```

Comment:
We should probably move these statements to a central location as high up the element chain as possible, because `<prism-highlighter>` captures all bubbled events. Bubbled events will bubble up all the way to `<rite-app>`